lib/models/BucketInfo.ts

```diff
@@ -240,6 +243,11 @@ export default class BucketInfo {
             READ_ACP: [],
         };
 
+        if (tags === undefined) {
+            tags = [] as BucketTag[];
+        }
+        validateTags(tags);
```

Comment:
Should this handle the return value? Calling this without handling the return seems weird.

Reply:
Thanks to your suggestion above, the new function can be passed to an assert to make sure it returns `true`: https://github.com/scality/Arsenal/pull/1933/commits/9c5365cca8d2dfc6445792f2b23aa14586bd8cb4#diff-4ee3cd5de98073dadb72b27f034f4979480c04388cc4f4538f4895373527f174R249